overlays/46-audio-receiver/usr/lib/systemd/system/kdeconnect.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=KDEConnect Daemon
+Documentation=https://kdeconnect.kde.org/
+After=network.target
+
+[Service]
+User=neon
+Environment="XDG_RUNTIME_DIR=/run/user/1004"
```

Comment:
Consider reading from an environment file like the core services (those use `/etc/neon/neon_env.conf`) since UID is not necessarily static